include/flash_map_pm.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#if USE_PARTITION_MANAGER
```

Comment:
Do we need this check inside this file? If this file is included, we already know that this is defined, right?